src/utils/src/validators.rs

```diff
@@ -2,30 +2,18 @@
 // SPDX-License-Identifier: Apache-2.0
 
 // Misc data format validations, shared by multiple Firecracker components.
-use std::fmt;
 
 const MAX_INSTANCE_ID_LEN: usize = 64;
 const MIN_INSTANCE_ID_LEN: usize = 1;
 
-#[derive(Debug, PartialEq, Eq)]
+#[derive(Debug, PartialEq, Eq, thiserror::Error)]
 pub enum Error {
-    InvalidChar(char, usize),        // (char, position)
+    #[error("invalid char ({0}) at position {1}")]
```

Comment:
I know it is out of this PR's scope, but could you please write this with capital letter? The final error message would look something like:
`Invalid instance ID: invalid char`
which is kind of ugly.

Reply:
Changed error message.
Is this OK?

Reply:
Yes.